deepinfra/models/base/automatic_speech_recognition.py

```diff
@@ -0,0 +1,35 @@
+from deepinfra.models.base import BaseModel
+from deepinfra.types.automatic_speech_recognition.request import AutomaticSpeechRecognitionRequest
+from deepinfra.types.automatic_speech_recognition.response import AutomaticSpeechRecognitionResponse
+from deepinfra.utils.form_data import FormDataUtils
+
+
+class AutomaticSpeechRecognition(BaseModel):
+
+	"""
+	Initializes the automatic speech recognition model.
+	@param endpoint: The endpoint of the model or the model name.
+	@param auth_token: The API key to authenticate the requests. If not provided, it will be fetched from the environment.
+	@docs Check the available models at https://deepinfra.com/models/automatic-speech-recognition
+	"""
+	def __init__(self, endpoint: str, auth_token: str = None):
+		super().__init__(endpoint, auth_token)
+		self.endpoint = endpoint
```

Comment:
The base model already assigns endpoint, you don't have to do it here. Same for auth_token